### PR TITLE
ENT-1620: Add option to use our cache of npmjs repository (Nexus)

### PR DESCRIPTION
**Pull request by jirihnidek**
* This commit tries to minimize trafic to npmjs repository from
  our project, because Red Hat has cache of this repository.
* Added two versions of yarn.lock file
  - yarn.lock.npmjs is intended for community users (default)
  - yarn.lock.nexus is for Red Hat developers
* Added symbolic link yarn.lock (it targets to yarn.lock.npmjs)
* Added instructions to README.md

### CODE REVIEW

**Comment by jirihnidek** [comment]
@candlepin-bot ok to test

**Comment by jirihnidek** [comment]
@candlepin-bot ok to test

**Comment by cnsnyder** [comment]
@candlepin-bot Please retest this
@jirihnidek Please rebase on master :)

File: cockpit/README.md [hunk 1]
```diff
@@ -13,6 +13,18 @@ code is edited.
...
 ------------------------------
+> Important note for Red Hat developers: When you try to develop cockpit plugin
+outside Red Hat office, then you needn't to do any changes of configuration file.
+In case you are Red Hat developer working in Red Hat office, then please use
+our internal cache of nmpjs repository called Nexus. All you need to do is change
+symbolic link: `rm -f yarn.lock; ln -s yarn.lock.nexus yarn.lock`. There are
+two lock files: `yarn.lock.npmjs` (for community users) and `yarn.lock.nexus` (for
+Red Hat developers). In case you did some big change in `yarn.lock.npmjs` and you
+want to have similar change in `yarn.lock.nexus`, then you can use following
+regul expression:
```
Comment (wottop) [inline cockpit/README.md]: Grammar cleanup:
 
An important note for Red Hat developers: If you develop the cockpit
plugin outside of the Red Hat office, no changes to the configuration file are
needed . If you are Red Hat developer working in the Red Hat office, then please use
our internal cache of the nmpjs repository called Nexus. You need to change the
symbolic link: `rm -f yarn.lock; ln -s yarn.lock.nexus yarn.lock`. There are
two lock files: `yarn.lock.npmjs` (for community users) and `yarn.lock.nexus` (for
Red Hat developers). If you make a big change in `yarn.lock.npmjs` and you
want to have similar change in `yarn.lock.nexus`, then you should use following
regular expression:

File: cockpit/README.md [hunk 2]
```diff
@@ -30,6 +42,11 @@ The most common commands are:
 
 (See https://yarnpkg.com for more details on using yarn).
 
+> When you are using our Nexus repository and you added some dependency using
+`yarn add <package>`, then please update `yarn.lock.npmjs` file. Vice verse, when
+you are community contributor and you added some package, then please update
+`yarn.lock.nexus` too.
+
```
Comment (wottop) [inline cockpit/README.md]: When using our Nexus repository and you add a dependency using
`yarn add <package>`, also update the `yarn.lock.npmjs` file. Conversely,
when you are community contributor and add a package, please update
`yarn.lock.nexus` as well.

File: cockpit/README.md (hunk 1 above)
Comment (jirihnidek) [reply cockpit/README.md]: Thanks!

File: cockpit/README.md (hunk 2 above)
Comment (jirihnidek) [reply cockpit/README.md]: Thanks

File: cockpit/README.md (hunk 1 above)
Comment (jirihnidek) [reply cockpit/README.md]: Thanks

**Comment by jirihnidek** [comment]
@wottop I updated `README.md`.

**Comment by wottop** [comment]
ack

**Pull request merged**